lib/components/FilePicker/FileList.vue

```diff
@@ -67,7 +67,7 @@
 						:key="file.fileid || file.path"
 						:allow-pick-directory="allowPickDirectory"
 						:show-checkbox="multiselect"
-						:can-pick="multiselect || selectedFiles.length === 0 || selectedFiles.includes(file)"
+						:can-pick="(canPick === undefined || canPick(file)) && (multiselect || selectedFiles.length === 0 || selectedFiles.includes(file))"
```

Comment:
Small performance improvement to not always execute the callback when not needed:


```suggestion
						:can-pick="(multiselect || selectedFiles.length === 0 || selectedFiles.includes(file)) && (canPick === undefined || canPick(file))"
```

So this way the cheap comparison is done on every file, but the potential expensive callback is only done when needed.

But this is theoretical - did not test it 😅

Reply:
Not sure if it makes that much sense. But applied ;)